ckanext/dashboard/actions/dashboard_dataset.py

```diff
@@ -125,14 +127,19 @@ def dataset_dashboard_delete(context, data_dict):
     """
     log.info("Executing dataset_dashboard_delete")
 
-    dashboard_id = toolkit.get_or_bust(data_dict, 'id')
+    dashboard_id = t.get_or_bust(data_dict, 'id')
```

Comment:
```suggestion
    dashboard_id = data_dict['id']
```
In this case, this is simpler and better